paymentsheet/src/main/java/com/stripe/android/common/ui/BottomSheet.kt

```diff
@@ -52,6 +56,10 @@ internal class BottomSheetState(
     }
 
     suspend fun hide() {
+        if (isRunningTest) {
```

Comment:
Not my finest moment, but I wasn’t able to figure out why `WindowInsets.isImeVisible` returns true in tests even when the keyboard has been closed 🙈